src/elements/content-uploader/__tests__/__snapshots__/ItemAction-test.js.snap

```diff
@@ -18,6 +18,7 @@ exports[`elements/content-uploader/ItemAction should render correctly with STATU
     theme="default"
   >
     <PlainButton
+      isDisabled={false}
```

Comment:
You shouldn't need to specify false

Reply:
This is the auto-generated snapshot. Should I edit it?

Reply:
Ah, never mind!